styles/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+    background-color: mistyrose;
+    font-family: Helvetica, sans-serif;
+    font: 100%;
+}
+
+.container{
```

Comment:
We usually write style rules with a space between the selector and the brace (as you did above), so
```css
.container {
```